Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -426,16 +467,47 @@ public class Fuzzer {
     ///   - program: The FuzzIL program to execute.
     ///   - timeout: The timeout after which to abort execution. If nil, the default timeout of this fuzzer will be used.
     /// - Returns: An Execution structure representing the execution outcome.
-    public func execute(_ program: Program, withTimeout timeout: UInt32? = nil) -> Execution {
+    public func execute(_ program: Program, withTimeout timeout: UInt32? = nil,
+                        differentialExecute: DifferentialExecute = .auto) -> Execution {
         dispatchPrecondition(condition: .onQueue(queue))
         assert(runner.isInitialized)
 
-        let script = lifter.lift(program)
+        let differentialResult: Bool
```

Comment:
Nit: maybe name this `performDifferentialTesting` or so? That'll make the `if` conditions below a bit more intuitive I think